apps/neuvector/neuvector/neuvector.yaml

```diff
@@ -97,7 +96,7 @@ spec:
                       values:
                         - system
         image:
-          repository: imported/neuvector/controller
+          repository: neuvector/controller
```

Comment:
you'll need to remove from here if you want the chart to work

Reply:
Can I revert this back for now to stop neuvector trying to build broken pods @timja ?

I can add in a separate PR with the chart version upgrade + image repository removals

Reply:
sure